datahub-web-react/src/app/entity/shared/tabs/Entity/components/EntityList.tsx

```diff
@@ -28,22 +35,90 @@ const StyledListItem = styled(List.Item)`
     padding-top: 20px;
 `;
 
+const PaginationInfoContainer = styled.span`
+    padding: 8px;
+    padding-left: 16px;
+    border-top: 1px solid;
+    border-color: ${(props) => props.theme.styles['border-color-base']};
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+`;
+
+const StyledPagination = styled(Pagination)`
+    margin: 0px;
+    padding: 0px;
+`;
+
+const PaginationInfo = styled(Typography.Text)`
+    padding: 0px;
+`;
+
 type EntityListProps = {
     type: EntityType;
     entities: Array<any>;
     title?: string;
+    totalJobs?: number | null;
+    pageSize?: any;
+    lastResultIndex?: any;
+    showTaskPagination?: boolean;
 };
 
-export const EntityList = ({ type, entities, title }: EntityListProps) => {
+export const EntityList = ({
+    type,
+    entities,
+    title,
+    totalJobs,
+    pageSize,
+    lastResultIndex,
+    showTaskPagination = false,
```

Comment:
`totalJobs` and `showTaskPagination` are a little too specific here - let's rename to `totalAssets` and `showPagination` to make this generic so we can use pagination in `EntityList` everywhere if we desire.

we then will probably need to pass in `page` and `setPage` from the parent so pagination can be handled outside of here and we don't have to rely on a pagination context